aiogram/client/session/base.py

```diff
@@ -183,7 +188,7 @@ async def __call__(
     ) -> TelegramType:
         middleware = partial(self.make_request, timeout=timeout)
         for m in reversed(self.middlewares):
-            middleware = partial(m, make_request=middleware)  # type: ignore
+            middleware = partial(m, middleware)  # type: ignore
```

Comment:
@JrooTJunior FYI, this changes is for compitability with RequestMiddlewareType, but this will break existing middlewares (if there is session middlewares used by users)

middleware signature changes from (bot, method, make_request) to (make_request, bot, method),

Reply:
I see that, i think it should be described in changelog

Reply:
Lgtm